planning/ThingDescription/td-next-work-items/usability-and-design.md

```diff
@@ -72,32 +72,92 @@ Related Issues:
   - The initial connection must not be mandatory to establish upon TD consumption and should left to the implementation when to establish and close it.
 
 **Notes:**
+
 - When to close the connection needs to be discussed.
 - Complex security mechanisms exchange should be handled at the same time
 
-#### Examples of Message Flow
+#### Examples of Message Sequences
```

Comment:
We should add text description to each lifecycle diagram explaining when the transitions happen.

Reply:
Explain what data is reused: In MQTT, client id is associated to a connection but is not really reused. In cases like OAuth2, the token is reused.

Reply:
Both points are addressed